qiskit/transpiler/passmanager.py

```diff
@@ -26,18 +28,27 @@
 
 
 class PassManager:
-    """A PassManager schedules the passes"""
+    """Manager for a set of Passes and their scheduling during transpilation."""
 
-    def __init__(self, passes=None, max_iteration=1000, callback=None):
-        """Initialize an empty PassManager object (with no passes scheduled).
+    def __init__(
+            self,
+            passes: Union[BasePass, List[BasePass]] = None,
+            max_iteration: int = 1000,
+            callback: Callable = None
+    ):
+        """Initialize an empty `PassManager` object (with no passes scheduled).
 
         Args:
-            passes (list[BasePass] or BasePass): A pass set (as defined in ``append()``)
-                to be added to the pass manager schedule. The default is None.
-            max_iteration (int): The schedule looping iterates until the condition is met or until
-                max_iteration is reached.
-            callback (func): DEPRECATED - A callback function that will be called after each
-                pass execution.
+            passes: A pass set (as defined in ``append()``) to be added to the pass manager
+                schedule.
+            max_iteration: The maximum number of iterations the schedule will be looped if the
+                condition is not met.
+            callback: DEPRECATED - A callback function that will be called after each pass
+                execution.
+
+        .. deprecated ::
+            The `callback` parameter is deprecated in favor of
```

Comment:
```suggestion
            The ``callback`` parameter is deprecated in favor of
```

Nice deprecation box!